pkg/trace/pb/remote_rates.go

```diff
@@ -0,0 +1,15 @@
+package pb
+
+// TargetTPS contains the targeted traces per second the agent should try to sample for a particular service and env
+type TargetTPS struct {
+	Service   string  `msgpack:"0"`
+	Env       string  `msgpack:"1"`
+	Value     float64 `msgpack:"2"`
```

Comment:
Can you please document these fields so that someone looking over the code can figure out what Value, Rank & Mechanism are? Otherwise PR looks good.

Reply:
documented the fields